test/hydration-test.js

```diff
@@ -103,6 +103,10 @@ describe('Hydration', function() {
     hydration = new Hydration(hydrationData);
   });
 
+  it('should only take in an array', function() {
```

Comment:
Good sad path test checking the data type. I wonder if we need to include the test to check in the array is not empty too.